libgssapi/src/error.rs

```diff
@@ -97,8 +101,8 @@ impl Error {
 
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        Error::fmt_code(f, self.major.bits(), "major")?;
-        Ok(Error::fmt_code(f, self.minor, "minor")?)
+        Error::fmt_code(f, self.major.bits(), GSS_C_GSS_CODE)?;
```

Comment:
Since there are only two possibilities for ctype we should make an enum, and then translate the enum into the C constant at the FFI call, that way we can eliminate the panic when later matching the value.

Reply:
Submitted. I can combine the commits and force push the result if you prefer a clean history.